registry.go

```diff
@@ -55,6 +55,15 @@ func Login(ctx context.Context, store Store, reg *remote.Registry, cred auth.Cre
 	return nil
 }
 
+// Logout provides the logout functionality given the registry name.
+func Logout(ctx context.Context, store Store, registryName string) error {
+	registryName = mapHostname(registryName)
+	if err := store.Delete(ctx, registryName); err != nil {
+		return fmt.Errorf("unable to logout of the registry %s: %w", registryName, err)
```

Comment:
This error message is not consistent with line 53.

BTW, for line 49, it should not be "unable to login to the registry" but "unable to ping the registry".

Reply:
changed accordingly.